src/pages/changePasswordPage/ResetPasswordPage.jsx

```diff
@@ -25,137 +29,177 @@ const ResetPasswordPage = () => {
     newPassword: false,
     confirmPassword: false,
   });
+
+  useEffect(() => {
+    const searchParams = new URLSearchParams(location.search);
+    const token = searchParams.get("token");
+    axios.get(`${API_END_POINT}/api/accounts/password/token_verification/?token=${token}`).then((res)=>{
+      if(res.data.message){
+        setVerificationToken(token);
+      }
+    }).catch((error)=>{
+      if(error.response && error.response.data.status === 400){
```

Comment:
Is there any other possibities of error other than 400?

Reply:
yes Anna if API is not working then the backend team gives two types of error messages it happens when 400 is coming so that's what I am using it

Reply:
There is no possibility of any error other than 400 Anna